hs/AboutResponse.hs

```diff
@@ -32,4 +34,5 @@ aboutHtml contents = H.div ! A.id "aboutDiv" $ mdToHTML contents
 -- | mdToHTML takes in the contents of a file written in Mark Down and converts it to 
 -- blaze-HTML.
 mdToHTML :: String -> H.Html
-mdToHTML contents = writeHtml def $ readMarkdown def contents
\ No newline at end of file
+mdToHTML contents = markdown def $ pack contents
```

Comment:
But also, I wouldn't mind if we used the `Data.Text.Lazy` `Text` type all the way through, to avoid the call to `pack` here. This would involve using `Data.Text.Lazy.IO`'s `readFile` in `server.hs`.